we know this isn't going to be in a git repo, so get rid of a useless dependency
on libgit2.

--- a/build.rs
+++ b/build.rs
@@ -1,12 +1,3 @@
-use vergen_git2::{Emitter, Git2Builder};
-
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    // Put "git describe" output (ex. "v0.5.0-2-gbc03f8a-dirty") in
-    // VERGEN_GIT_DESCRIBE environment variable for use in --version output.
-    let describe = Git2Builder::default()
-        .describe(true, true, Some("v[0-9]*.[0-9]*.[0-9]*"))
-        .build()?;
-    Emitter::default().add_instructions(&describe)?.emit()?;
-
     Ok(())
 }
